bql/planner/planner_test.go

```diff
@@ -1107,6 +1107,26 @@ func TestPlannerQuery(t *testing.T) {
 			nBindings: 2,
 			nRows:     1,
 		},
+		{
+			q: `SELECT ?p, ?o
+				FROM ?test
+				WHERE {
+					/u<peter> ?p ?o .
+					FILTER isImmutable(?p)
+				};`,
+			nBindings: 2,
+			nRows:     2,
+		},
+		{
+			q: `SELECT ?s, ?p, ?o
+				FROM ?test
+				WHERE {
+					?s ?p ?o .
+					FILTER isImmutable(?o)
+				};`,
+			nBindings: 3,
+			nRows:     1,
+		},
```

Comment:
Can you add some tests considering aliases as well?

Reply:
Sure, thank you for the comment Thiago, I have just done so. =]